helpdesk/management/commands/get_email.py

```diff
@@ -100,7 +137,11 @@ def process_queue(q, quiet=False):
         socks.set_default_proxy(proxy_type=proxy_type, addr=q.socks_proxy_host, port=q.socks_proxy_port)
         socket.socket = socks.socksocket
     else:
-        socket.socket = socket._socketobject
+        if six.PY2:
```

Comment:
Is setting the socket to _socketobject necessary at all? I can't find documentation about this anywhere, and this might be some legacy code..
Are you aware of how the socket.socket object is used in the remainder?